flake.lock: Update

hrp2-14 opened the pull request:
Automated changes by the [update-flake-lock](https://github.com/DeterminateSystems/update-flake-lock) GitHub Action.

```
Flake lock file updates:

• Updated input 'agimus-msgs':
    'github:agimus-project/agimus_msgs/a0a2ebadf42341a3f0eb0f61b54ac717a00ca2ed?narHash=sha256-iNxwSrKZhN%2BfG5Mg8bIL7xWiX/Ss3OkJAsx3hNRYLoo%3D' (2025-01-24)
  → 'github:agimus-project/agimus_msgs/689a413e93c697e092bd8ac69a27a23230f07fe4?narHash=sha256-HOcglC0O1AEZFvqk6RqQB4wiC43z%2BlOjp1ILLlZaVf4%3D' (2025-02-12)
• Updated input 'colmpc':
    'github:agimus-project/colmpc/cad9284735d7308e19f7027ecb7bb101d90a0297?narHash=sha256-cUa3sV1k2MzXfDD9BeWBa4Hz33MllRRqMDp17U3POkE%3D' (2025-01-23)
  → 'github:agimus-project/colmpc/96ed40c157f290b8541a52e675ae195e1a07bb3c?narHash=sha256-14NqWLY%2BJWxOiYE2d4QTyBvOHzfoX3Q55UOlru2zwZc%3D' (2025-03-03)
• Added input 'colmpc/crocoddyl':
    'github:loco-3d/crocoddyl/871b0a933f1f74f0abb8bbbce0d367c769eeb435?narHash=sha256-2xceLlDCaUxUWMoCZralxsBwxwBc2jhNK7iqccIz7rU%3D' (2025-02-07)
• Added input 'colmpc/crocoddyl/flake-parts':
    follows 'colmpc/flake-parts'
• Added input 'colmpc/crocoddyl/nixpkgs':
    follows 'colmpc/nixpkgs'
• Updated input 'colmpc/flake-parts':
    'github:hercules-ci/flake-parts/205b12d8b7cd4802fbcb8e8ef6a0f1408781a4f9?narHash=sha256-4pDvzqnegAfRkPwO3wmwBhVi/Sye1mzps0zHWYnP88c%3D' (2024-12-04)
  → 'github:hercules-ci/flake-parts/32ea77a06711b758da0ad9bd6a844c5740a87abd?narHash=sha256-7H9XgNiGLKN1G1CgRh0vUL4AheZSYzPm%2BzmZ7vxbJdo%3D' (2025-02-01)
• Updated input 'colmpc/flake-parts/nixpkgs-lib':
    'https://github.com/NixOS/nixpkgs/archive/5487e69da40cbd611ab2cadee0b4637225f7cfae.tar.gz?narHash=sha256-1qRH7uAUsyQI7R1Uwl4T%2BXvdNv778H0Nb5njNrqvylY%3D' (2024-12-01)
  → 'https://github.com/NixOS/nixpkgs/archive/072a6db25e947df2f31aab9eccd0ab75d5b2da11.tar.gz?narHash=sha256-vJzFZGaCpnmo7I6i416HaBLpC%2BhvcURh/BQwROcGIp8%3D' (2025-02-01)
• Updated input 'franka-description':
    'github:agimus-project/franka_description/ea3edc76581a7e02be6e28624f711f76dcbee25a?narHash=sha256-96Bwca5WqajRUVYKntDKMWvTNEwaoZcl/nVS5YLM1V8%3D' (2025-02-05)
  → 'github:agimus-project/franka_description/819ad93f90012fcb33a2051e7b4bdf3b2ac37a09?narHash=sha256-PsfLYDtSXK01jHitjOo4thLlL%2BFjr1m8592V8Ar2VNQ%3D' (2025-02-05)
• Updated input 'linear-feedback-controller-msgs':
    'github:loco-3d/linear-feedback-controller-msgs/e7a1db1c1a2024978116af29db568606046fe547?narHash=sha256-X3SHjw9FBs/Z8M11P9SSUZf/o2FpR8ttzs7vdEwq8yk%3D' (2025-01-28)
  → 'github:loco-3d/linear-feedback-controller-msgs/cf9d1d249f9ffd80d9092847eb7a298f7ce0f49c?narHash=sha256-Qz99AM39QSrG0n5SoF32YdjIVKWixOgQkwaHezbrmzY%3D' (2025-02-11)
• Updated input 'nix-ros-overlay':
    'github:lopsided98/nix-ros-overlay/f9c218f6e94b59d037bb081ba3cf99e2b567ee49?narHash=sha256-4/4AJHCJiReTb%2BQtYubUncxiD%2BtUkz8ehBvhi81dcOE%3D' (2025-02-01)
  → 'github:lopsided98/nix-ros-overlay/902905a45b5f3b2aa2771bf5c59df1fcfe003778?narHash=sha256-8nCsAs0KB7kb8kkgX3zVS8M5zJqbnH2pzlAZPsnknOE%3D' (2025-03-01)
• Updated input 'nix-ros-overlay/nixpkgs':
    'github:lopsided98/nixpkgs/22c3f2cf41a0e70184334a958e6b124fb0ce3e01?narHash=sha256-Qn3nPMSopRQJgmvHzVqPcE3I03zJyl8cSbgnnltfFDY%3D' (2024-12-07)
  → 'github:lopsided98/nixpkgs/8bb37161a0488b89830168b81c48aed11569cb93?narHash=sha256-%2B/bSz4EAVbqz8/HsIGLroF8aNaO8bLRL7WfACN%2B24g4%3D' (2025-02-15)
• Updated input 'patch-hpp':
    'https://github.com/nim65s/nixpkgs/pull/2.patch?narHash=sha256-H3FDc6QA501TMwwnWuLdpudnEiyPvDWnuMazZ3pNsdE%3D'
  → 'https://github.com/nim65s/nixpkgs/pull/2.patch?narHash=sha256-Rj07W99G9CvZEc/92lAJa8JYqgFKhaIoeyJuatpLlhk%3D'
```

### Running GitHub Actions on this PR

GitHub Actions will not run workflows on pull requests which are opened by a GitHub Action.

To run GitHub Actions workflows on this PR, run:

```sh
git branch -D update_flake_lock_action
git fetch origin
git checkout update_flake_lock_action
git commit --amend --no-edit
git push origin update_flake_lock_action --force
```

nim65s commented:
```
python3.12-agimus-controller> E   ImportError: /nix/store/kylq46k3qj9wc0hm2h406fwh3h3qcnwa-mim-solvers-0.0.5/lib/python3.12/site-packages/mim_solvers/../../../../lib/libmim_solvers.so.0.0.5: undefined symbol: _ZN9crocoddyl14SolverAbstractC2EN5boost10shared_ptrINS_18ShootingProblemTplIdEEEE

github-actions[bot] committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'agimus-msgs':
    'github:agimus-project/agimus_msgs/a0a2ebadf42341a3f0eb0f61b54ac717a00ca2ed?narHash=sha256-iNxwSrKZhN%2BfG5Mg8bIL7xWiX/Ss3OkJAsx3hNRYLoo%3D' (2025-01-24)
  → 'github:agimus-project/agimus_msgs/689a413e93c697e092bd8ac69a27a23230f07fe4?narHash=sha256-HOcglC0O1AEZFvqk6RqQB4wiC43z%2BlOjp1ILLlZaVf4%3D' (2025-02-12)
• Updated input 'colmpc':
    'github:agimus-project/colmpc/cad9284735d7308e19f7027ecb7bb101d90a0297?narHash=sha256-cUa3sV1k2MzXfDD9BeWBa4Hz33MllRRqMDp17U3POkE%3D' (2025-01-23)
  → 'github:agimus-project/colmpc/96ed40c157f290b8541a52e675ae195e1a07bb3c?narHash=sha256-14NqWLY%2BJWxOiYE2d4QTyBvOHzfoX3Q55UOlru2zwZc%3D' (2025-03-03)
• Added input 'colmpc/crocoddyl':
    'github:loco-3d/crocoddyl/871b0a933f1f74f0abb8bbbce0d367c769eeb435?narHash=sha256-2xceLlDCaUxUWMoCZralxsBwxwBc2jhNK7iqccIz7rU%3D' (2025-02-07)
• Added input 'colmpc/crocoddyl/flake-parts':
    follows 'colmpc/flake-parts'
• Added input 'colmpc/crocoddyl/nixpkgs':
    follows 'colmpc/nixpkgs'
• Updated input 'colmpc/flake-parts':
    'github:hercules-ci/flake-parts/205b12d8b7cd4802fbcb8e8ef6a0f1408781a4f9?narHash=sha256-4pDvzqnegAfRkPwO3wmwBhVi/Sye1mzps0zHWYnP88c%3D' (2024-12-04)
  → 'github:hercules-ci/flake-parts/32ea77a06711b758da0ad9bd6a844c5740a87abd?narHash=sha256-7H9XgNiGLKN1G1CgRh0vUL4AheZSYzPm%2BzmZ7vxbJdo%3D' (2025-02-01)
• Updated input 'colmpc/flake-parts/nixpkgs-lib':
    'https://github.com/NixOS/nixpkgs/archive/5487e69da40cbd611ab2cadee0b4637225f7cfae.tar.gz?narHash=sha256-1qRH7uAUsyQI7R1Uwl4T%2BXvdNv778H0Nb5njNrqvylY%3D' (2024-12-01)
  → 'https://github.com/NixOS/nixpkgs/archive/072a6db25e947df2f31aab9eccd0ab75d5b2da11.tar.gz?narHash=sha256-vJzFZGaCpnmo7I6i416HaBLpC%2BhvcURh/BQwROcGIp8%3D' (2025-02-01)
• Updated input 'franka-description':
    'github:agimus-project/franka_description/ea3edc76581a7e02be6e28624f711f76dcbee25a?narHash=sha256-96Bwca5WqajRUVYKntDKMWvTNEwaoZcl/nVS5YLM1V8%3D' (2025-02-05)
  → 'github:agimus-project/franka_description/819ad93f90012fcb33a2051e7b4bdf3b2ac37a09?narHash=sha256-PsfLYDtSXK01jHitjOo4thLlL%2BFjr1m8592V8Ar2VNQ%3D' (2025-02-05)
• Updated input 'linear-feedback-controller-msgs':
    'github:loco-3d/linear-feedback-controller-msgs/e7a1db1c1a2024978116af29db568606046fe547?narHash=sha256-X3SHjw9FBs/Z8M11P9SSUZf/o2FpR8ttzs7vdEwq8yk%3D' (2025-01-28)
  → 'github:loco-3d/linear-feedback-controller-msgs/cf9d1d249f9ffd80d9092847eb7a298f7ce0f49c?narHash=sha256-Qz99AM39QSrG0n5SoF32YdjIVKWixOgQkwaHezbrmzY%3D' (2025-02-11)
• Updated input 'nix-ros-overlay':
    'github:lopsided98/nix-ros-overlay/f9c218f6e94b59d037bb081ba3cf99e2b567ee49?narHash=sha256-4/4AJHCJiReTb%2BQtYubUncxiD%2BtUkz8ehBvhi81dcOE%3D' (2025-02-01)
  → 'github:lopsided98/nix-ros-overlay/902905a45b5f3b2aa2771bf5c59df1fcfe003778?narHash=sha256-8nCsAs0KB7kb8kkgX3zVS8M5zJqbnH2pzlAZPsnknOE%3D' (2025-03-01)
• Updated input 'nix-ros-overlay/nixpkgs':
    'github:lopsided98/nixpkgs/22c3f2cf41a0e70184334a958e6b124fb0ce3e01?narHash=sha256-Qn3nPMSopRQJgmvHzVqPcE3I03zJyl8cSbgnnltfFDY%3D' (2024-12-07)
  → 'github:lopsided98/nixpkgs/8bb37161a0488b89830168b81c48aed11569cb93?narHash=sha256-%2B/bSz4EAVbqz8/HsIGLroF8aNaO8bLRL7WfACN%2B24g4%3D' (2025-02-15)
• Updated input 'patch-hpp':
    'https://github.com/nim65s/nixpkgs/pull/2.patch?narHash=sha256-H3FDc6QA501TMwwnWuLdpudnEiyPvDWnuMazZ3pNsdE%3D'
  → 'https://github.com/nim65s/nixpkgs/pull/2.patch?narHash=sha256-Rj07W99G9CvZEc/92lAJa8JYqgFKhaIoeyJuatpLlhk%3D'

MaximilienNaveau reviewed (approved):
Let's focus on fixing the code in another PR.
